### ISSUE TITLE
Windows: Hovering over the meters of an unfocussed project window brings that window to the front - original project retains focus

### ISSUE THREAD

**Issue by petersampsonaudacity** [audacity/audacity#4094]
### Bug description

On Windows, hovering over the Meter toolbars of an unfocussed project window brings that window to the front, however the original project (properly) retains focus.

Testing shows that this is a regression on 2.1.2 - _in earlier versions hovering over the Mixer toolbar produces the bug_

### Steps to reproduce

1. Launch Audacity
2. get some audio - Project-1
3. Ctrl+N to launch a second project - Project-2
4. get some audio in Project-2
5. arrange the window so that both projects have their meter toolbars on-screen
6. Click on Project-1 to make it active
7. Press Play
8. Hover your cursor over the meter toolbars in Project-2
9. Observe: Project-2 becomes the front window - and Project-1 is still playing (i.e. Project-1 has focus)

### Expected behavior

At Steps 8-9 Project-1 should remain the top window

### Actual behavior

Project2 becomes the top window however Project-1 is actively playing i.e. has focu

![image](https://user-images.githubusercontent.com/50205984/208912823-a0547893-eb4a-4037-a0ef-a5e355eb881c.png)

### Audacity Version
3.2.2, 3.2.3 Beta and 3.3.0 alpha

### Operating system
Windows 10 

Tested on macOS 12.6.1 and this does not affect Mac

### Additional context

Originally reported on the Forum by user mj10008:
**Unwanted switches between multiple open projects**
https://forum.audacityteam.org/viewtopic.php?f=46&t=128351&p=461174

**Comment by petersampsonaudacity** [comment]
Still occurs in 3.2.4

see this Forum post:
**I think this is a bug involving volume and balance sliders**
https://forum.audacityteam.org/viewtopic.php?f=46&t=129211

**Comment by petersampsonaudacity** [comment]
Also reported here on 3.2.5 by a Forum user:

**Ver. 3.2.5 : bug when switching window**
https://forum.audacityteam.org/viewtopic.php?f=46&t=129537

**Comment by Ryan-Miller-Audacity** [comment]
Tested and reproduced this behavior on Windows 10 in Audacity 3.3.2 Release.

**Comment by kevintimba** [comment]
3 observations:

1. I've experienced this bug for months, but these steps don't reproduce it when starting from scratch. The program seems to need to run for a while to get into the state where the bug occurs. 
2. I agree that it's connected to the volume and balance sliders.
3. The program doesn't need to be playing for the bug to occur

Possible Cause:  If Window 1 is in focus, but Window 2's volume sliders are visible, and I click on one of them to adjust it, it adjusts, but Window 1 remains in focus. I'm not sure if it's a "bug or feature" that the focus doesn't switch completely to Window 2. It seems like clicking on Window 2's slider should bring the entirely of Window 2 into focus (but see below). To look at this a different way, I'm typing this comment into Firefox, and I have a File Explorer window open elsewhere on my screen. If I hover my mouse over the File Explorer window without clicking, the Firefox window remains in focus. I can (as I would expect to be able to) continue to type, even though the mouse pointer is hovering over the File Explorer window. But if I roll my mouse wheel, the contents of the File Explorer window scroll. I can scroll those contents up and down with my mouse hand while continuing to type this comment with my other hand on the keyboard. This has nothing to do with Audacity so wouldn't we say that it's acceptable or maybe desirable to be able to move the sliders in the not-in-focus Audacity window?

There's still another counter-intuitive Audacity behavior related to this. If you hold down the control key and roll the mouse while while hovering over different parts of an Audacity track, sometimes the vertical zoom gets affected in bizarre ways. All of this is connected. We need to get some genius who fully understands the etiquette of how multiple windows are meant to work with the mouse in Windows. The expensive corporate programs that we're hoping Audacity will equal and surpass have a standardized etiquette such that this family of annoyingly weird Audacity behaviors don't occur in those programs. A possible approach to fixing this longstanding family of bugs would be to compare the behavior of Audacity to Pro Tools et al and make Audacity work that way.

In summary: it's not just this one bug - it's a family of bugs with this one root cause of not following the standard protocol for focus and mouse wheel behavior with multiple windows. Especially with these new time stretching features, Audacity is becoming such a wonderful program that there's all the more reason to clean this up!

**Comment by LWinterberg** [comment]
Fwiw: We're in the process of redoing the UI from scratch. Fixing this particular bug isn't a high priority because of that - none of these behaviors are likely to survive on the way over.

**Comment by kevintimba** [comment]
> Fwiw: We're in the process of redoing the UI from scratch.

Wow - that's exciting news! Any ETA?

**Comment by LWinterberg** [comment]
Hello! We just [switched development to Audacity 4](https://forum.audacityteam.org/t/master-has-been-switched-to-4-0-0-development/120444). We've redone, or plan on redoing, various things from scratch, including the entire UI, and the part this bug is about. As such, I'm closing this bug as it will be fixed there and we won't be fixing it for the final releases of Audacity 3.

**Comment by kevintimba** [comment]
Thanks very much for clarifying. Any estimates for the first stable version
of Audacity 4? And will it support the new openVINO AI stem separation?
That's a massive improvement!
